test/index.js

```diff
@@ -88,10 +88,10 @@ testRule(selectorBemPattern.rule, {
   ],
 });
 
-testRule(selectorBemPattern.rule, {
-  ruleName: selectorBemPattern.ruleName,
+testRule({
+  plugins: ['.'],
+  ruleName,
   config: null,
-  skipBasicChecks: true,
```

Comment:
Moving to `jest-preset-stylelint` has been pretty straightforward.

`skipBasicChecks` has been removed in older jest-preset-stylelint version https://github.com/stylelint/jest-preset-stylelint/blob/main/CHANGELOG.md#500